curriculum/challenges/english/25-front-end-development/lab-leap-year-calculator/66c06fad3475cd92421b9ac2.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 66c06fad3475cd92421b9ac2
+title: Build a Leap Year Calculator
+challengeType: 14
+dashedName: build-a-leap-year-calculator
+---
+
+# --description--
```

Comment:
I vote we add a description here on what a leap year actually since some people might not know.